<launch>

  <node pkg="earth_rover_localization"  name="sbp_arbitrator" type="sbp_arbitrator.py" output="log">
      <!-- Ntrip params are loaded from autonomous rover farm.yaml-->
      <!-- Radio params -->
      <param name="radio_port" value="/dev/freewaveGXMT14" />
      <param name="radio_baudrate" value="115200" />
      <!-- Udp params -->
      <param name="udp_address" value="192.168.8.222" />
      <param name="udp_port" value="55558" />
      <!-- Main loop frequency [Hz] -->
      <param name="frequency" value="5" />
      <!-- Enable debugging options -->
      <param name="debug" value="True" />
    </node>

</launch>
